validation/js/src/main/scala/hmda/validation/rules/ts/syntactical/S028.scala

```diff
@@ -0,0 +1,15 @@
+package hmda.validation.rules.ts.syntactical
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.dsl.{ Result, HmdaDsl, CommonDsl }
```

Comment:
We noticed that the js and jvm versions of these files are identical, except for the order of imports. Is there any way to move this code to the `shared` directory? (We see that each one depends on the respective version of date-related code, but we don't know whether that dependency prevents sharing the code.)